src/main/java/com/digitalpebble/storm/crawler/bolt/ParserBolt.java

```diff
@@ -300,8 +305,17 @@ public void execute(Tuple tuple) {
             }
         }
 
-        collector.emit(tuple, new Values(url, content, metadata, text.trim(),
-                slinks));
+        for (String outlink : slinks) {
+            // configure which metadata gets inherited from parent
+            Map<String, String[]> linkMetadata = metadataTransfer
+                    .getMetaForOutlink(metadata);
+            collector
+                    .emit(com.digitalpebble.storm.crawler.Constants.StatusStreamName,
+                            tuple, new Values(outlink, linkMetadata,
```

Comment:
Well I don't have anything in particular in mind. But I'm wondering how you will implement the click path that we want for our project and maybe the page URL could be handy